engine/source/engine.cpp

```diff
@@ -33,6 +35,9 @@
 
 #include <time_module.hpp>
 
+SoundInfo si;
```

Comment:
So how are sounds tracked by the engine? It might be more intuitive to pass the sound infos to the audio module so they are loaded, and the user gets a SoundHandle returned. This also avoids someone loading the sound and then changing the path or any properties after loading.

The sound handle is then used for calling sounds and updating 3D values and such

Reply:
This is a better idea yea, I have implemented it thank you  👍